src/openvic-simulation/economy/ProductionType.cpp

```diff
@@ -96,7 +94,14 @@ bool ProductionTypeManager::add_production_type(
 		return false;
 	}
 
-	if (template_type == ProductionType::template_type_t::ARTISAN) {
+	using enum ProductionType::template_type_t;
+
+	if (template_type == ARTISAN) {
+		if (owner.get_pop_type() != nullptr && owner.get_pop_type()->get_is_artisan()) {
```

Comment:
Discusssed on discord.
Specifying any pop here is meaningless and silently accepting it can create the impression that it does have an effect.
For modders: either leave out the owner scope or leave 'artisan'* as vanilla.

*technically any name that is not a pop type is fine.

Reply:
Since owners don't do anything in vanilla and should be removed from there too, e.g. if someone is making a mod based off of vanilla, I think the warning should apply for any owner being given, regardless of whether the pop type is `artisan`, a real pop type, or any other key.